packages/ui/src/1_atoms/ErrorBadge/ErrorBadge.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from 'react';
+
+import classNames from 'classnames';
+
+import { applyDataAttr } from '../../utils';
+import styles from './ErrorBadge.module.css';
+import { ErrorData } from './ErrorBadge.types';
+import { getIcon } from './ErrorBadge.utils';
+
+export type ErrorBadgeProps = ErrorData;
+
+export const ErrorBadge: FC<ErrorBadgeProps> = ({
```

Comment:
Realised we still have plain Div used in FastBTC flow - it wasn't in original scope but it should be a quick change, can you update the following locations to use ErrorBadge instead?
![image](https://user-images.githubusercontent.com/765825/219692792-b2e52630-468a-45fe-949b-4411923cc2c3.png)